code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -448,6 +449,12 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/omnizine/protozine
```

Comment:
How do you get this, dont see it made anywhere

Reply:
it was added here to be used in diluted syringes but I decided to not add the reagent used in those so it's maint pill fodder currently